app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const { psqlErrors, customErrors, handle500s } = require("./errors/index");
+const { getTopics } = require("./controllers/topics.controllers.js");
+const app = express();
+app.use(express.json());
```

Comment:
This isn't being used yet, it's only needed when a request has a body

Reply:
Understood.